contrib/templates/app-template/README.md

```diff
@@ -0,0 +1,10 @@
+# app-template template
+
+This template contains a bundle structure to create and manage Databricks App which reads data from a SQL warehouse and visualizes it.
```

Comment:
```suggestion
This template contains a bundle structure for creating and managing a Databricks app that reads data from a SQL warehouse and visualizes it.
```

Looking at https://docs.databricks.com/en/dev-tools/databricks-apps/index.html, the capitalization rules appear to be "Databricks Apps" as the product name but a "Databricks app" when talking about a single instance.